magprime/templates/static_views/secrets.html

```diff
@@ -0,0 +1,14 @@
+<!doctype html>
+<html>
+<head>
+    <title> Secrets...</title>
+</head>
+<body>
+
+<h3 align="center"> Secrets... </h3>
+
+<p>Since MAGFest doesn't sell advertising or sponsorship, all of our income comes from the support our extremely generous community of attendees. For those of you who want to kick in as much as you can to support MAGFest's mission, we want to put something special together to show our appreciation. You'll have to wait and see what it is, but we promise it'll show you how important you are to our existence. If for some reason you're insane enough that you want to give us MORE money for MORE stuff, purchasing this badge upgrade will put you on a list to be invited to exclusive additional upgrades later in the year.</p>
```

Comment:
"Exclusive additional upgrades later in the year"

Hissssssss

Obviously this isn't code-related feedback, but the Registration department SHOULD really be warned before you promise our time to people. Like ya do here.